### ISSUE TITLE
Add Apache Phoenix 5.2.1 Docker Image build to ghcr.io

### ISSUE THREAD

**Issue by linghengqian** [Boostport/hbase-phoenix-all-in-one#5]
- I noticed that https://issues.apache.org/jira/browse/PHOENIX-6919 and https://issues.apache.org/jira/browse/PHOENIX-6912 have fallen silent, so I expect that the current git will add a Docker Image build of Apache Phoenix 5.2.1 on ghcr.io .
- I'm not sure if apache phoenix itself is using https://github.com/cloudera/opdb-docker or https://github.com/Boostport/hbase-phoenix-all-in-one for integration testing, but both git seem to use an older version of phoenix for builds.

**Comment by F21** [comment]
@linghengqian Yes, it's a bit unfortunate that there has been no movement for the jira issues you linked. I myself have not worked with Phoenix for quite a while and so, I am a bit unfamiliar with that is required to created a docker image containing the latest version of phoenix. Would you be happy to contribute an up-to-date docker image to the ASF Phoenix repository? I think it would be very welcomed!

**Comment by linghengqian** [comment]
> Would you be happy to contribute an up-to-date docker image to the ASF Phoenix repository? 
- Honestly, I probably can't help. As a shardingsphere committer, I'm looking for a docker image of Apache Phoenix for integration testing at https://github.com/apache/shardingsphere . But I actually know nothing about how Apache Phoenix works.
